### PR TITLE
erase backup if the user does not want to restore

### PR DESCRIPTION
**Pull request by ThatNerdyPikachu**
should i add another option to not restore and keep the backup?

*that's probably a better idea :/*

### CODE REVIEW

**Comment by thehappydinoa** [comment]
Hey, I think we should consider leaving the backup there as other launchers backup there too

**Comment by joeyhuab** [comment]
That's actually a really good idea.

**Comment by ThatNerdyPikachu** [comment]
I'll make some changes, give me a sec.

**Comment by ThatNerdyPikachu** [comment]
Wait.. we already remove it?

File: common/install.sh [hunk 1]
```diff
@@ -197,5 +197,6 @@ if [ -f /data/media/0/.launcher.db.backup ] && [ -z $NORESTORE ]; then
   else
     ui_print " "
     ui_print "   Did not restore!"
+    rm -rf /data/media/0/.launcher.db.backup
```
Comment (thehappydinoa) [inline common/install.sh]: Would you consider adding another:

```bash
ui_print " - Erase Backup -"
ui_print "   Do you want to erase?"	     
ui_print "   Vol+ = Keep backup, Vol- = ERASE backup"
if $FUNCTION; then
    ui_print " "
    ui_print "   Keeping backup"
else
    ui_print " "
    ui_print "   Deleting backup"
	rm -rf /data/media/0/.launcher.db.backup
fi
```

**Comment by ThatNerdyPikachu** [comment]
See if those changes matched your thoughts @thehappydinoa.

If not, I'd be happy to adjust anything.

**Review by thehappydinoa — COMMENTED** [review]
This looks good to me, have you tested it?

**Comment by ThatNerdyPikachu** [comment]
I have not gotten the chance to, does anyone mind testing the last 2 changes?

**Comment by joeyhuab** [comment]
Am not on my device since it's charging :(

**Comment by joeyhuab** [comment]
Maybe ask someone in the tele group: https://t.me/PIX3LIFY

**Comment by thehappydinoa** [comment]
Ya, I cannot either as I don't use the Pixel Launcher

**Comment by ThatNerdyPikachu** [comment]
Going to hold on merging until we fix the launcher bug that may affect this.

**Review by joeyhuab — APPROVED** [review]
Will add this in. Thanks!

**Pull request merged**